plugins/aws/iam/trustedCrossAccountRoles.js

```diff
@@ -0,0 +1,96 @@
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Trusted Cross Account Roles',
+    category: 'IAM',
+    description: 'Ensures that only trusted cross-account IAM roles can be used.',
+    more_info: 'IAM roles should be configured to allow access to trusted account IDs.',
+    link: 'https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_common-scenarios_aws-accounts.html',
+    recommended_action: 'Delete the IAM roles that are associated with untrusted account IDs.',
+    apis: ['IAM:listRoles'],
+    settings: {
+        whitelisted_accounts: {
+            name: 'Allowed cross-account role account IDs',
+            description: 'Return a failing result if cross-account role contains any account ID other than these account IDs',
+            regex: '^[0-9{12}]$',
```

Comment:
This should be extended so that the commas can be handled. For example:
Valid: `123456789101`
Valid: `123456789101,123456789102,123456789103`
NOT Valid: `123456789101,`
NOT Valid: `123456789101,,,`
```suggestion
            regex: '^[0-9{12}]$',
```